files/default/sql/ddl/updates/3.3.0.sql

```diff
@@ -238,3 +238,7 @@ ALTER TABLE `hopsworks`.`feature_store_statistic`
 
 -- Git executions hostname
 ALTER TABLE `hopsworks`.`git_executions` ADD COLUMN `hostname` VARCHAR(128) COLLATE latin1_general_cs NOT NULL DEFAULT "localhost";
+
+-- FSTORE-857
+UPDATE hopsworks.jobs SET json_config=JSON_SET(json_config, '$.appPath', 'hdfs:///user/spark/hsfs-utils-3.3.0-RC0.jar') WHERE JSON_EXTRACT(json_config, '$.appPath') like '"hdfs:///user/spark/hsfs-utils-%.jar"';
```

Comment:
I think you need to disable safe updates here because you don't have a where condition on the PK